libminifi/test/opencv-tests/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+#
```

Comment:
Thanks for adding a test, can this be moved into the opencv extensions folder?

Reply:
Not sure I follow on this ask. You have an example you are thinking of? I basically just followed what the TensorFlow one was doing for this. Honestly had no idea if the location was correct or not.